pkg/sql/sem/builtins/builtins.go

```diff
@@ -4848,6 +4852,75 @@ var jsonArrayLengthImpl = tree.Overload{
 	Volatility: tree.VolatilityImmutable,
 }
 
+func messageToJSON(name string, data []byte) (json.JSON, error) {
+	// Get the reflected type of the protocol message.
+	rt := proto.MessageType(name)
```

Comment:
this is kind of magic -- is there some centralized source of all registered proto types?